AutomatedTesting/Gem/PythonTests/AWS/common/aws_utils.py

```diff
@@ -16,7 +16,7 @@
 class AwsUtils:
 
     def __init__(self, arn: str, session_name: str, region_name: str):
-        local_session = boto3.Session(profile_name='default')
+        local_session = boto3.Session()

```

Comment:
Surprised there aren't changes to the README.md over how to setup your credentials? Esp if we previously just took the default profile.

Reply:
Sure. I can update README for the AWS credentials.